floor_generator/lib/writer/query_method_writer.dart

```diff
@@ -137,11 +138,15 @@ class QueryMethodWriter implements Writer {
     @nullable final String arguments,
     @nonNull final String mapper,
   ) {
-    final entityName = _queryMethod.queryable.name;
-
+    final tableName = _queryMethod.queryable.name;
+    final isView =
+        _queryMethod.queryable.runtimeType == View ? 'true' : 'false';
```

Comment:
💡Dart comes with `is` as a language feature. You can simplify this code with the following.
```suggestion
    final isView = _queryMethod.queryable is View ? 'true' : 'false';
```

❓Did you look into `bool`'s `toString()`? Is there the need for explicitly defining 'true' and 'false' instead of just invoking `isView.toString()`? Do you think it's better to be explicit here?

Reply:
no, toString seems good here. I'll change the code.